boards/lilygo/ttgo_t8c3/ttgo_t8c3.dts

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright 2024 Lothar Felten <lothar.felten@gmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <espressif/esp32c3/esp32c3_common.dtsi>
+#include "ttgo_t8c3-pinctrl.dtsi"
+
+/ {
+	model = "Lilygo ttgo t8-c3";
```

Comment:
needs uppercase / official name here too, you might want to do a global search/replace